arch/arm/configs/bcm2711_defconfig

```diff
@@ -514,6 +514,7 @@ CONFIG_BCMGENET=y
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
 CONFIG_QCA7000_UART=m
+CONFIG_R8169=m
```

Comment:
These changes were merged independently yesterday - see #4625.

Reply:
Yes, that snuck with fetching latest changes from upstream. I'll drop this commit from PR when I push the changes for README and overlay.